src/utils/gif_utils.js

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export function generateUUID() {
```

Comment:
We've got a similar `generateId` function in `utils/helpers.js` to this. Would this be able to use that instead?

Reply:
Yes.  I've refactored to use.